packages/shared-lib/src/package-manager/helpers.ts

```diff
@@ -0,0 +1,61 @@
+import { ExpectedPackage } from './package'
+
+// Note: These functions are copied from Package Manager
+
+type Steps = Required<ExpectedPackage.ExpectedPackageHtmlTemplate['version']>['steps']
+
+export function htmlTemplateGetSteps(version: ExpectedPackage.ExpectedPackageHtmlTemplate['version']): Steps {
+	let steps: Steps
+	if (version.casparCG) {
+		// Generate a set of steps for standard CasparCG templates
+		const casparData = version.casparCG.data
+		const casparDataJSON = typeof casparData === 'string' ? casparData : JSON.stringify(casparData)
+		steps = [
+			{ do: 'waitForLoad' },
+			{ do: 'takeScreenshot', fileName: 'idle.png' },
+			{ do: 'startRecording', fileName: 'preview.webm' },
+			{ do: 'executeJs', js: `update(${casparDataJSON})` },
+			{ do: 'executeJs', js: `play()` },
+			{ do: 'sleep', duration: 1000 },
+			{ do: 'takeScreenshot', fileName: 'play.png' },
```

Comment:
perhaps if any steps are manually defined they should also be used?
either inserted after this screenshot, or replacing everything except the update?

Reply:
A good point, though I don't have any good idea for how to define this in a not-too-messy way. We could always add this later.